src/components/registerpage/registerHelpers.js

```diff
@@ -0,0 +1,4 @@
+export function validateEmail(email) {
+  const re = /^[\w-\.]+@([\w-]+\.)+[\w-]{2,4}$/;
```

Comment:
src/components/register -> move all files, you dont need registerpage. also pages generally not components

as a future note, but this is fine for now, you never want to write your own regexes for emails. go google regex for valid emails and you'll see why. this regex probably isn't correct.

Reply:
Is [this](https://emailregex.com/) valid?

Reply:
PTAL